### PR TITLE
Issue #710 Reorganize minimum system requirements

### PR DESCRIPTION
**Pull request by robin-owen**

### CODE REVIEW

File: docs/source/getting-started/content/ref_minimum-system-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{prod} requires the following system resources:
+
+* 4 virtual CPUs (vCPUs)
+* 8 GB of memory
+* 35 GB of storage space
+
+[NOTE]
+====
+These requirements must be met in order to run OpenShift in the {prod} virtual machine.
+Other applications running on top of OpenShift may require additional resources to run.
```
Comment (gbraad) [inline docs/source/getting-started/content/ref_minimum-system-requirements.adoc]: This is actually for sure needed. 'Depending on the workload more resources need to be assigned'

File: docs/source/getting-started/content/ref_minimum-system-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+[id="minimum-system-requirements-operating-system_{context}"]
+== Operating system requirements
+
+{prod} requires the following minimum version of a supported operating system:
+
+=== {msw}
+
+* On {msw}, {prod} requires the Windows 10 Fall Creators Update (version 1709).
+{prod} does not work on earlier versions of {msw}.
```
Comment (gbraad) [inline docs/source/getting-started/content/ref_minimum-system-requirements.adoc]: Mention: `Windows 10 Pro/Home`, `does not work on earlier versions or editions of {msw}`

**Comment by gbraad** [comment]
@teuf and/or @praveenkumar PTAL

File: docs/source/getting-started/content/ref_minimum-system-requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{prod} requires the following minimum hardware and operating system requirements.
+
+[id="minimum-system-requirements-hardware_{context}"]
+== Hardware requirements
+
+{prod} requires the following system resources:
+
+* 4 virtual CPUs (vCPUs)
+* 8 GB of memory
+* 35 GB of storage space
```
Comment (praveenkumar) [inline docs/source/getting-started/content/ref_minimum-system-requirements.adoc]: 👍

**Pull request merged**